### ISSUE TITLE
Accessibility keywords are not respected

### ISSUE THREAD

**Issue by BoBoBaSs84** [BoBoBaSs84/BB84.SourceGenerators#15]
**Describe the bug**

Accessibility keywords are not respected, when choosing internal or private for the abstraction source generator.

**To Reproduce**

1. Use the attribute
2. Make the implementation class internal 
3. Generate the code

**Expected behavior**

Accessibility keywords should be respected.

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Version [1.2.409.0]
